src/Data/List/Relation/Unary/AllPairs.agda

```diff
@@ -18,7 +18,7 @@ open import Function.Base using (id; _∘_)
 open import Level using (_⊔_)
 open import Relation.Binary.Definitions as B
 open import Relation.Binary.Construct.Intersection renaming (_∩_ to _∩ᵇ_)
-open import Relation.Binary.PropositionalEquality
+open import Relation.Binary.PropositionalEquality.Core using (refl; cong₂)
```

Comment:
Fine. I guess that the current decomposition of the `PropositionalEquality` modules is pretty much set in stone for the time being, so the localisation makes sense.